`add an option to pay out estimated rewards instead of final rewards (#47)
cryptiumlabs/backerei: README.md

* `--payout-delay -1` will pay the rewards one cycle (three days) before you actually
get access to them

> You can not set a payout delay of less than negative `PRESERVED_CYCLES` because final

Reviewer: ```suggestion
You cannot set a payout delay of less than negative `PRESERVED_CYCLES` because final
```
